README.md

```diff
@@ -23,3 +23,18 @@ inefficient.
 
 What is the big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+## Answer:
+
+We know that when it comes to finding the closest nodes in this code and finding the closest unvisited node and finding its distance will be $O|V|$ time. It is also executed V times leading to us getting $O|V^2|$ for the time in finding the closest edge. 
+
+Our code also has to update the distances once they have been found by the neighboring node. This time would be $O(E)$. E is the edges in the code. 
+
+The time complexity for the vertices would be $O|V^2|$. Everything combined it would be $O((V^2)+E)$ or $O((|E| + |V|)log|V|)$
+
+
+## Sources:
+I watched this video on Dijstra's algorithm so I could understand the time complexity better than what was given to me in the slides and the video provided to us. https://www.youtube.com/watch?v=YMyO-yZMQ6g. I also watched this video on how to write the code in JAVA to get a better understanding of how to write it because I was a little lost on how to implement infinity into the code and do the loops for updating the distance (https://www.youtube.com/watch?v=q3yKyE19OR0). I also looked at Caden's repo to figure out how to the test code correctly and pass. 
```

Comment:
What part of the code did you write yourself?

Reply:
I updated it with comments. It should be all good now. Let me know if you have any questions about it.